test/one_hot_encoding.jl

```diff
@@ -35,8 +35,8 @@
             @test FeatureTransforms.apply(x, ohe; inds=2:4) == [0 0 1; 0 1 0; 0 0 1]
             @test FeatureTransforms.apply(x, ohe; dims=:) == expected
 
-            @test_throws DimensionMismatch FeatureTransforms.apply(x, ohe; dims=1)
```

Comment:
`dims` no longer takes slices in `apply` so it plays no part in `OneHotEncoding` unless used with `inds`